.rubocop.yml

```diff
@@ -6,23 +6,26 @@ AllCops:
   NewCops: enable
 
 Metrics/BlockLength:
-  Exclude: &fastfiles
+  # "xfiles" is a standin for `Fast-`, `Pod-`, and `Rake-file`
```

Comment:
<img alt="The truth is out there" src="https://user-images.githubusercontent.com/216089/173823945-c85fbede-8d55-4b50-a743-3b428149db9b.png" width="300px" />